src/storage/admin/StatisTask.cpp

```diff
@@ -131,17 +131,15 @@ StatisTask::genSubTask(GraphSpaceID spaceId,
         edgetypeEdges[edge.first] = 0;
     }
 
-    TagID                                 lastTagId = 0;
     VertexID                              lastVertexId = "";
-    // Only statis valid vetex data
+    // Only statis valid vetex data, no multi version
     // For example
-    // Vid  tagId  version
-    // 1     1     2
-    // 1     1     1
-    // 2     2     2   (invalid data)
-    // 2     2     1   (invalid data)
-    // 2     3     1
-    // 3     1     2
+    // Vid  tagId
+    // 1     1
+    // 2     2  (invalid data)
```

Comment:
Tag 2 is dropped? why invalid, better add some comments.

Reply:
Has added comments.
invalid tag data, for example, tag data without tag schema.  create tag, insert tag data, then drop tag, so this tag data is invalid.